shards.nix

```diff
@@ -0,0 +1,7 @@
+{
+  db = {
+    url = "https://github.com/crystal-lang/crystal-db.git";
+    rev = "v0.11.0";
```

Comment:
What's the command for bumping this

Reply:
If you are in the dev shell (either with `nix develop` or (what I do) have an .envrc file of `use flake`) then it's `crystal2nix` 

If not I think `nix develop -c crystal2nix` will just create a temp shell with crystal2nix in the path and then drop out of it right away